Info.lua

```diff
@@ -1096,6 +1132,12 @@ g_PluginInfo =
 			RecommendedGroups = "admins",
 		},
 
+		["core.gamerule"] =
+		{
+			Description = "Allows players to set and query game rule values",
+			RecommendedGroups = "admins",
+		},
+
```

Comment:
@madmaxoft Should I have `core.gamerule` allow setting and querying,
`core.gamerule.query` for querying and listing only, and
`core.gamerule.set` for setting only _or_ setting **and** querying?